CONTRIBUTING.md

```diff
@@ -17,3 +17,16 @@ like this:
 ```
 export TOX_PYTHON_27=~/.pyenv/versions/2.7.14/bin/python
 ```
+
+## Local configuration
+You can configure the test server with the following variables:
+### DASH_TEST_CHROMEPATH
+If you run a special chrome set the path to your chrome binary with this environment variable.
+
+### DASH_TEST_PROCESSES
+If you encounter errors about Multi-server + Multi-processing when running under Python 3 try running the tests with the number of server processes set to 1.
```

Comment:
A couple of tests in the `dash-renderer` repo depend on there being multiple processes, in particular: https://github.com/plotly/dash-renderer/blob/c0f886d2a6d2507c22527a86be614f3c2838b667/tests/test_render.py#L1360-L1366

In those tests, one callback is supposed to hang while another process serves the other callback. I don't think that any tests in this repo absolutely require multiple processes.

So, we'll just have to be cautious about adding it to `dash-renderer` and we'll need to make sure that CI still runs things with multiple processes